addons/l10n_it_edi_ndd/models/account_move.py

```diff
@@ -19,6 +20,17 @@ class AccountMove(models.Model):
         readonly=False,
     )
 
+    def _auto_init(self):
+        """
+        Create compute stored field l10n_it_document_type and l10n_it_payment_method
+        here to avoid timeout error on large databases.
+        """
```

Comment:
This is not information about the method but rather why you use it, it shouldn't be docstring but rather simple comment lines :+1:
```suggestion
        # Create l10n_it_document_type and l10n_it_payment_method stored computed columns
        # here to avoid timeouts in large databases
```